App/src/test/java/cc/softwarefactory/lokki/android/espresso/utilities/TestUtils.java

```diff
@@ -66,4 +69,35 @@ public static void toggleNavigationDrawer() {
         // TODO: hardcoded click position and menu text
         onView(withId(R.id.decor_content_parent)).perform(clickScreenPosition(0, 0));
     }
+
+    public static ViewAction swipeScreen(final int x, final int y, final int z, final int w) {
```

Comment:
I'd rather use parameter names like x1, y1, x2, y2 (or fromX, fromY, toX, toY) since z and w sounds like they are another axes, which they are not.